week2/index.js

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+const argumentArray = process.argv;
+const command = process.argv[2];
+const argument = process.argv[3];
+switch (command) {
+  case "help":
+  case undefined:
+    fs.readFile('help.txt', 'utf8', (err, data) => {
+      if (err) throw err;
+      console.log(data);
+    });
+    break;
+  case "add":
+    fs.appendFile('todoList.txt', "\n" + argument, (err) => {
+      if (err) throw err;
+      console.log('The task is added successfully');
+    });
+    break;
+    case "remove":
+    fs.readFile('todoList.txt', 'utf8', (err, data) => {
+      if (err) throw err;
+      const dataToArray = data.split("\n");
+      dataToArray.splice(argument, 1);
```

Comment:
Homework specifies that we should `remove a todo item by its 1-base index`. Here you are using the 0-index. Which means that if we execute `node index.js remove 1` the second todo item will be removed, while we wanted the first todo item to be removed. How can we fix that?